x/ccv/consumer/keeper/distribution.go

```diff
@@ -16,6 +16,11 @@ import (
 
 const TransferTimeDelay = 1 * 7 * 24 * time.Hour // 1 weeks
 
+// The fraction of tokens allocated to the consumer redistribution address
+// during distribution events. The fraction is a string representing a
+// decimal number. For example "0.5" would represent 50%.
+const ConsumerRedistributeFrac = "0.5"
```

Comment:
@jtremback Do you think we should set this to 0.75 as the default value?

Reply:
So one thing to mention is that because this is a const it's "testing unfriendly" basically we have to use whatever is in here as the testing value (50% is simple to reason about). This being said, previously we'd set the default to 0% but we need a value higher than 0% to test functionality properly... so yeah, we should probably just have notes about parameterizing this value in some form of "master" guide for those who want to use CCV

Reply:
I agree. We need documentation on how to bootstrap a consumer chain (there is an open issue for this https://github.com/cosmos/interchain-security/issues/54).

Reply:
```suggestion
const ConsumerRedistributeFrac = "0.75"
```